{"EventType":"NewsyDbContext","EntityFrameworkEvent":{"Database":"newsy","ConnectionId":"beef50a7-e3d0-4caa-be7f-e7cab033e82f","ContextId":"04e35142-0ffe-4e2b-81bd-bdcb90684359:0","TransactionId":"beef50a7-e3d0-4caa-be7f-e7cab033e82f_253403070466","Entries":[{"Table":"AppUsers","Name":"AppUser","PrimaryKey":{"Id":"2d40e9a3-8fb6-4cc8-b5a0-0db638b12219"},"Action":"Insert","ColumnValues":{"Id":"2d40e9a3-8fb6-4cc8-b5a0-0db638b12219","AccessFailedCount":0,"Active":false,"Archived":false,"ArchivingDate":null,"ConcurrencyStamp":"c1f06eaf-ce2b-4773-9833-2f90009ecb9f","CreationDate":"2021-09-26T21:19:15.10773+02:00","Email":"b@p.com","EmailConfirmed":false,"Enabled":false,"FirstName":"Pero","LastName":"Peroc","LastPasswordChangeDateTime":"0001-01-01T00:00:00","LockoutEnabled":false,"LockoutEnd":null,"NormalizedEmail":"B@P.COM","NormalizedUserName":"B@P.COM","PasswordHash":"AQAAAAEAACcQAAAAEJSzJtjnscsWh97IJMEqpPn1M3yPU7l3Z942sOU/+scX6L6k/NiIoswn/NhoF5YW1w==","PhoneNumber":null,"PhoneNumberConfirmed":false,"SecurityStamp":null,"TwoFactorEnabled":false,"UserName":"b@p.com"},"Valid":true},{"Table":"AppUserRoles","Name":"AppUserRole","PrimaryKey":{"UserId":"2d40e9a3-8fb6-4cc8-b5a0-0db638b12219","RoleId":"49ec873f-76a1-4ba4-87cd-63ba1316438f"},"Action":"Insert","ColumnValues":{"UserId":"2d40e9a3-8fb6-4cc8-b5a0-0db638b12219","RoleId":"49ec873f-76a1-4ba4-87cd-63ba1316438f"},"Valid":true}],"Result":0,"Success":false,"ErrorMessage":"Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details.\r\n ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Cannot insert duplicate key row in object 'dbo.AppUsers' with unique index 'UserNameIndex'. The duplicate key value is (B@P.COM).\r\nThe statement has been terminated.\r\n   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__169_0(Task`1 result)\r\n   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()\r\n   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)\r\n   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\nClientConnectionId:beef50a7-e3d0-4caa-be7f-e7cab033e82f\r\nError Number:2601,State:1,Class:14\r\n   --- End of inner exception stack trace ---\r\n   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(DbContext _, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\r\n   at Audit.EntityFramework.DbContextHelper.SaveChangesAsync(IAuditDbContext context, Func`1 baseSaveChanges)"},"Environment":{"UserName":"Brunc","MachineName":"DESKTOP-N7RN9HE","DomainName":"DESKTOP-N7RN9HE","CallingMethodName":"Audit.EntityFramework.DbContextHelper+<CreateAuditScopeAsync>d__31.MoveNext()","AssemblyName":"Audit.EntityFramework.Core, Version=18.1.5.0, Culture=neutral, PublicKeyToken=571d6b80b242c87e","Exception":"COMException: 0xE0434352","Culture":"en-GB"},"StartDate":"2021-09-26T19:19:20.4842941Z","EndDate":"2021-09-26T19:19:20.5883766Z","Duration":104}